hotham/src/shaders/pbr.frag

```diff
@@ -305,43 +307,22 @@ void main()
 				outColor.rgba = baseColor;
 				break;
 			case 2:
+				vec3 n = getNormal(material.normalTextureID);
 				outColor.rgb = n * 0.5 + 0.5;
 				break;
 			case 3:
-				outColor.rgb = (material.occlusionTextureID == NO_TEXTURE) ? vec3(0.0f) : texture(textures[material.occlusionTextureID], inUV).ggg;
+				outColor.rgb = (material.occlusionTextureID == NOT_PRESENT) ? ERROR_MAGENTA.rgb : texture(textures[material.occlusionTextureID], inUV).ggg;
 				break;
 			case 4:
-				outColor.rgb = (material.emissiveTextureID == NO_TEXTURE) ?  vec3(0.0f) : texture(textures[material.emissiveTextureID], inUV).rgb;
+				outColor.rgb = (material.emissiveTextureID == NOT_PRESENT) ?  ERROR_MAGENTA.rgb : texture(textures[material.emissiveTextureID], inUV).rgb;
 				break;
 			case 5:
-				outColor.rgb = (material.physicalDescriptorTextureID == NO_TEXTURE) ? vec3(0.0) : texture(textures[material.physicalDescriptorTextureID], inUV).ggg;
+				outColor.rgb = (material.physicalDescriptorTextureID == NOT_PRESENT) ? ERROR_MAGENTA.rgb : texture(textures[material.physicalDescriptorTextureID], inUV).ggg;
 				break;
 			case 6:
-				outColor.rgb = (material.physicalDescriptorTextureID == NO_TEXTURE) ? vec3(0.0) : texture(textures[material.physicalDescriptorTextureID], inUV).aaa;
+				outColor.rgb = (material.physicalDescriptorTextureID == NOT_PRESENT) ? ERROR_MAGENTA.rgb : texture(textures[material.physicalDescriptorTextureID], inUV).aaa;
 				break;
 		}
 		outColor = outColor;
 	}
-
-	// "none", "Diff (l,n)", "F (l,h)", "G (l,v,h)", "D (h)", "Specular"
-	if (sceneData.params.w > 0.0) {
-		int index = int(sceneData.params.w);
-		switch (index) {
-			case 1:
-				outColor.rgb = diffuseContrib;
-				break;
-			case 2:
-				outColor.rgb = F;
-				break;
-			case 3:
-				outColor.rgb = vec3(G);
-				break;
-			case 4:
-				outColor.rgb = vec3(D);
-				break;
-			case 5:
-				outColor.rgb = specContrib;
-				break;
-		}
-	}
```

Comment:
Yeah I couldn't figure out a way to keep them without really breaking the shader. But in hindsight a much better way to test the renderer is by matching the full renders of a diverse set of models (especially adversarial ones like the `NormalMirrorTangent` test) rather than getting hyper specific on a single model.